scripts/build-ffmpeg.py

```diff
@@ -7,16 +8,22 @@
 
 from cibuildpkg import Builder, Package, get_platform, log_group, run
 
-if len(sys.argv) < 2:
-    sys.stderr.write("Usage: build-ffmpeg.py <prefix> [stage]\n")
-    sys.stderr.write(
-        "       AArch64 build requires stage and possible values can be 1, 2 or 3\n"
-    )
-    sys.exit(1)
+parser = argparse.ArgumentParser("build-ffmpeg")
+parser.add_argument("destination")
+parser.add_argument(
+    "--stage",
+    default=None,
+    help="AArch64 build requires stage and possible values can be 1, 2 or 3",
+)
+parser.add_argument("--disable-gpl", action="store_true")
+args = parser.parse_args()
 
-dest_dir = sys.argv[1]
-output_dir = os.path.abspath("output")
+dest_dir = args.destination
+build_stage = None if args.stage is None else int(args.stage) - 1
+disable_gpl = args.disable_gpl
+del args
```

Comment:
This is very unusual for Python code, why the `del`?

Reply:
`del` is used to signify that args will not be used anymore